### PR TITLE
refactor(console): show paywall tag and beta notice for mfa

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

update product per design

## Testing

image

image

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [x] necessary TSDoc comments

### CODE REVIEW

File: packages/console/src/ds-components/CardTitle/index.tsx [hunk 1]
```diff
@@ -20,6 +20,12 @@ export type Props = {
   className?: string;
   /** If a beta tag should be shown next to the title. */
   isBeta?: boolean;
+  /**
+   * If a paywall tag should be shown next to the title. The value is the plan type.
+   *
+   * If not provided, no paywall tag will be shown.
+   */
+  paywall?: 'hobby' | 'pro';
```
Comment (charIeszhao) [inline packages/console/src/ds-components/CardTitle/index.tsx:28]: nit: I think we do have an enum for plans. Might consider referencing these values from the plan enum for the sake of SSOT?

**Review by charIeszhao — APPROVED** [review]

File: packages/console/src/ds-components/CardTitle/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/ds-components/CardTitle/index.tsx:28]: @xiaoyijun any sugguestions?

**Review by wangsijie — APPROVED** [review]

File: packages/console/src/ds-components/CardTitle/index.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/ds-components/CardTitle/index.tsx:28]: We have a `ReservedPlanId` enum.

Comment (gao-sun) [reply packages/console/src/ds-components/CardTitle/index.tsx:28]: let me take a look

**Pull request merged**